server/src/unifycr_sock.c

```diff
@@ -101,7 +101,7 @@ int sock_init_server(void)
     poll_set[0].events = POLLIN | POLLHUP;
     poll_set[0].revents = 0;
     num_fds++;
-    printf("completed sock init server\n");
+    LOG(LOG_DBG, "completed sock init server\n");
```

Comment:
The LOG macro will add the \n, so let's drop that too from these new LOG lines.

Reply:
will do